spk/python38/Makefile

```
@@ -0,0 +1,46 @@
+SPK_NAME = python38
+SPK_SHORT_VERS = 3.8
+SPK_VERS = $(SPK_SHORT_VERS).7
+SPK_REV = 1
+SPK_ICON = src/python3.png
+
+DEPENDS  = cross/$(SPK_NAME)
+DEPENDS += cross/setuptools cross/pip cross/wheel
+DEPENDS += cross/cffi cross/bcrypt cross/sqlite
+# Cross-compiled wheels
+DEPENDS += cross/lxml cross/pycrypto cross/pycurl cross/pyyaml
+DEPENDS += cross/msgpack-python cross/ruamel.yaml cross/immutables
+DEPENDS += cross/cryptography
+
+WHEELS = src/requirements.txt
+
+MAINTAINER = Allan Clark
+DESCRIPTION = Python Programming Language.
+DESCRIPTION_FRE = Langage de programmation Python.
+DESCRIPTION_SPN = Lenguaje de programación Python.
+RELOAD_UI = yes
+STARTABLE = no
+DISPLAY_NAME = Python 3.8
+CHANGELOG = "Initial Python 3.8.7 release"
```

as the package is python38:
```suggestion
CHANGELOG = "Initial Python 3.8 release"
```

---

(Just realized you had proposed a change... I did it manually instead :/ )